CONTRIBUTING.md

```diff
@@ -266,6 +266,41 @@ For managing includes in C files, use [include-what-you-use].
 
 See [#549][549] for more details.
 
+Documenting
```

Comment:
isn't this redundant with `:help dev-doc` ? idk how this found its way to the wiki

Reply:
No objection to merging this with `dev-doc`; the separation feels arbitrary anyway. (Personally, I find the markdown files easier to read, especially when they make use of links to web pages.)